package.json

```diff
@@ -46,13 +43,18 @@
     "h264-profile-level-id": "^1.0.0",
     "netstring": "^0.3.0",
     "random-number": "^0.0.9",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "gulp": "^4.0.2",
+    "gulp-clang-format": "^1.0.27",    
+    "gulp-eslint": "^6.0.0",
```

Comment:
As far as I see we do not need `gulp-eslint`, `gulp-rename` and `gulp-touch-cmd` at all. They are not used anywhere in this PR.

Also, `gulp-clang-format` must remain in "devDependencies".

Regarding `gulp` in "dependencies" let's discuss the topic about gulpfile as mentioned above.

Reply:
unused deps already removed.